test/Market.t.sol

```diff
@@ -629,31 +661,50 @@ contract Market_SANCTION_ALLOWLIST_ROLE is UpgradeableMarket {
   }
 }
 
-contract Market__isAuthorizedWithdrawal_true is NonUpgradeableMarket {
+contract Market__isAuthorizedWithdrawal_true is
+  NonUpgradeableMarket,
+  UpgradeableRemoval
+{
+  Removal private _mockRemoval;
```

Comment:
since the function changed to now make a call to `_removal.hasRole.(...)` (which itself calls `_removal.CONSIGNOR_ROLE()` we needed to make the mock for these return values more robust and explicit than the basic mock that was in place for `isApprovedForAll` which is what this is about